COMPILING.md

```diff
@@ -0,0 +1,161 @@
+
+Compile the sources to ignore the JACK sound library:
+
+~~~
+qmake "CONFIG+=nosound headless" Jamulus.pro
+make clean
+Make
+~~~
+
+To control the server with systemd, see this [unit file example](https://github.com/jamulussoftware/jamulus/blob/master/distributions/jamulus-server.service). See also runtime [configuration options](/wiki/Command-Line-Options), and [this information](/wiki/Tips-Tricks-More#controlling-recording-on-linux-headless-servers) on controlling recordings on headless servers.
+
+### Notes
+
+* Compiling with the headless flag means you can avoid installing some of the dependent packages, save some disk space and/or speed up your build time under the following circumstances:
+
+* If you plan to run Jamulus on Gentoo Linux, the only packages you should need for a headless build are `qtcore`, `qtnetwork`, `qtconcurrent` and `qtxml` (both for building and running the server).
+
+* If you are running Jamulus on Ubuntu/Debian, you will need all dependent packages to compile the binary, but to run the resulting headless Jamulus server you should only need `libqt5core5a`, `libqt5network5`, `libqt5xml5` and probably `libqt5concurrent5`. This may be useful for compiling/upgrading on one machine to run the binary on another (a Raspberry Pi, for example).
+
+* Note that if you want to compile a GUI client on one machine and run it on another (e.g. a Raspberry Pi) you only need the dependencies listed for a headless server, only with the JACK sound libraries.
+
+---
+
+## Windows
+
+
+You will need [Qt](https://www.qt.io/download)
+
+* Use the free GPLv2 license for Open Source development
+* To determine the Qt version you need, check [qt-installer-windows.qs](https://github.com/jamulussoftware/jamulus/blob/master/windows/qt-installer-windows.qs): under INSTALL_COMPONENTS you will see `qt.qt5.[version]`, e.g., 5123 means version 5.12.3.
+* Select Components during installation: Expand the Qt section, find the matching version, e.g., **Qt 5.12.3**, and add the compiler components for your compiler, e.g., `MSVC 2017 32-bit/64-bit` for Visual Studio 2019
+* [ASIO development files](https://www.steinberg.net/en/company/developer.html)
+
+### Compiling and building installer
+
+Most users will probably want to use this method:
+
+1. Open PowerShell
+1. Navigate to the `jamulus` directory
+1. To allow unsigned scripts, right-click on the `windows\deploy_windows.ps1` script, select properties and allow the execution of this script. You can also run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. (You can revert this after having run this script. For more information see the [Microsoft PowerShell documentation page](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy)).
```

Comment:
This seems to mix PowerShell (console) and Windows Explorer, doesn't it?

Reply:
I've not altered any of this since it went into the original. Not sure who wrote it in fact. Maybe @ann0see ?

Reply:
Yes. I think I wrote it and somebody else edited it later. Not sure who it was. See my comment above.